js/holmes.js

```diff
@@ -86,42 +86,45 @@
      * Options
      * @type {Object}
      */
-    holmes.prototype.options = options;
+    holmes.prototype.options = {
+      input: 'input[type=search]',
+      placeholder: '',
+      placeholderNode: null,
+      mark: false,
+      class: {
+        visible: '',
+        hidden: 'hidden'
+      },
+      dynamic: false,
+      minCharacters: 0,
+      find: ''
+    };
+
 
     // if holmes.prototype.options.find is missing, the searching won't work so we'll thrown an exceptions
     if (typeof holmes.prototype.options.find !== 'string') {
       throw new Error('A find argument is needed. That should be a querySelectorAll for each of the items you want to match individually. You should have something like: \nholmes({\n\tfind:".result"\n});\nsee also https://haroen.me/holmes/doc/module-holmes.html');
     }
 
+    holmes.prototype.mergeObj = function (Obj1, Obj2) {
+      var k;
+      for(k in Obj2){
```

Comment:
For ... in is hard to do properly, and the linter doesn't allow it: 

`  117:18  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins`

Can you refactor this into a `forEach`?